craft.lic

```diff
@@ -144,6 +144,7 @@ class Craft
       order_item(crafting_data['artificing'][@hometown]['stock-room'], 2)
       order_item(crafting_data['artificing'][@hometown]['stock-room'], 6)
       stow_hands
+      walk_to(@training_room)
```

Comment:
Why remove this from craft.lic? You only want to move the movement logic into workorders right? Craft does movement  on its own for every other craft